src/CMakeLists.txt

```diff
@@ -9,6 +8,42 @@ include_directories("md/inc")
 include_directories("classlibnative/bcltype")
 include_directories("classlibnative/cryptography")
 include_directories("classlibnative/inc")
+include_directories("${GENERATED_INCLUDE_DIR}")
+
+set (GeneratingEventFilesCommand)
+if(WIN32)
+    list (APPEND GeneratingEventFilesCommand
+          COMMAND ${MC} -h ${GENERATED_INCLUDE_DIR} -r ${GENERATED_INCLUDE_DIR} -b -co -um -p FireEtw "${VM_DIR}/ClrEtwAll.man"
```

Comment:
Ah, I've missed the fact that the MC invocation is another difference in the command. 
So if I may suggest to restructure it a bit differently, I'd prefer it to be this way:
1) Move the MC invocation to a separate command that you would add only for WIN32, let its output be the ClrEtwAll.h (and ClrEtwAll.cpp - not necessary, but for completeness).
2) Put the `COMMAND ${PYTHON} -B -Wall ...` back to the common add_custom_command, parametrizing only the python script
3) Set a variable with the python script path in the if / else
4) Remove the ClrEtwAll.h output from the common custom command